lib/Installation/Partitioner/LibstorageNG/v4_3/ExpertPartitionerPage.pm

```diff
@@ -0,0 +1,98 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: The class introduces all accessing methods for Expert Partitioner
+# Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package Installation::Partitioner::LibstorageNG::v4_3::ExpertPartitionerPage;
+use strict;
+use warnings;
+use testapi;
+use parent 'Installation::Partitioner::LibstorageNG::ExpertPartitionerPage';
+
+use YuiRestClient;
+
+sub new {
+    my ($class, $args) = @_;
+    my $self = bless {
+        app => $args->{app}
+    }, $class;
+
+    return $self->init();
+}
```

Comment:
@rwx788  why you do not call the `YuiRestClient::get_app()` directly here? you can avoid to pass the argument every time in the controller. Actually it looks like it could initialized in the `Installation::Partitioner::LibstorageNG::ExpertPartitionerPage` or maybe to the super that all the Pages are inherited from.

Reply:
> Nice work man. Another thing i was thinking is if you even need the Page Object pattern anymore

Thanks. It's actually the opposite, Page Object pattern is now much more suitable.

Reply:
@jknphy Has already mentioned it and yeah, we should create base page class as long as we have common parts.
`YuiRestClient::get_app()` is less flexible and was the solution focused on the installation scenario where we have single app. But in future we might have multiple instances running.